snuba/datasets/storages/spans_v2.py

```diff
@@ -0,0 +1,63 @@
+from snuba.clickhouse.columns import UUID, ColumnSet, DateTime, Float
+from snuba.clickhouse.columns import SchemaModifiers as Modifiers
+from snuba.clickhouse.columns import String, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.datasets.schemas.tables import WritableTableSchema
+from snuba.datasets.spans_processor import SpansMessageProcessorV2
+from snuba.datasets.storage import WritableTableStorage
+from snuba.datasets.storages import StorageKey
+from snuba.datasets.table_storage import build_kafka_stream_loader_from_settings
+from snuba.query.processors.type_converters.hexint_column_processor import (
+    HexIntColumnProcessor,
+)
+from snuba.utils.streams.topics import Topic
+from snuba.web.split import TimeSplitQueryStrategy
+
+columns = ColumnSet(
+    [
+        ("project_id", UInt(64)),
+        ("transaction_id", UUID()),
+        ("trace_id", UUID()),
+        ("transaction_span_id", UInt(64)),
+        ("span_id", UInt(64)),
+        ("parent_span_id", UInt(64, Modifiers(nullable=True))),
+        ("transaction_name", String()),
+        ("span_group_id", UUID()),
```

Comment:
Clickhouse doesn't have an `UInt128` type so I'm using `UUID` here because the `span_group_id` is currently an `MD5` hash. Though we could potentially change this to a `UInt64` and choose a different hash function.

Reply:
UUID makes sense to me. This is the grouping hash that's based on description amongst other things?

Reply:
This grouping hash is currently based on a normalized description. Of course, we can add more to it to refine the grouping as needed. The relevant PR is getsentry/sentry#28184

Reply:
Why do we need a full 128 bit? 
https://en.wikipedia.org/wiki/Birthday_attack
With 64 bits you will have less than 0.1% chances of collision with a 10^8 different values. And even if it happened, this is a similarity indicator, occasional  collision would likely be harmless.

Why is this important: 128 bit  makes it a column twice as big as a 64 bit one. Which means it has a 2x impact on the data to scan, directly impacting the response time of your query.